foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/util/DataLineageUtil.java

```diff
@@ -0,0 +1,74 @@
+package com.boozallen.aissemble.upgrade.util;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class DataLineageUtil {
+
+    public static String VERSION_AISSEMBLE_POST_1_7_0 = "1.7.0";
+
+    public static final String VERSION_AISSEMBLE_POST_1_7_0_REGEX = "com\\.boozallen\\.aissemble\\.data\\.lineage";
+
+    public static final String VERSION_AISSEMBLE_PRE_1_7_0_REGEX = "com\\.boozallen\\.aiops\\.data\\.lineage";
```

Comment:
S: There's no reason for this to be regex since we're matching literal characters.  I'd just do literal string based searching instead.

Reply:
Updated the logic such that it does a literal string search (removed regex pattern matching).